config/action.d/mail-whois-lines.conf

```diff
@@ -4,6 +4,10 @@
 # Modified-By: Yaroslav Halchenko to include grepping on IP over log files
 #
 
+[INCLUDES]
+
+before = mail-whois-common.conf
```

Comment:
ok -- let's leave as is since switching between those might be trickier than just overriding a variable with .local file.  I am ok to keep save default, but also ok to make default more powerful and rely on systems having those core tools.